src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/JudgmentValidationService.java

```diff
@@ -23,7 +23,8 @@ public void validateJudgments(CaseData caseData) throws ParseException {
     }
 
     private void populateJudgmentDateOfHearing(JudgementType judgementType) throws ParseException {
-        if (NO.equals(judgementType.getNonHearingJudgment())) {
+        if (NO.equals(judgementType.getNonHearingJudgment())
+                && !"No Hearings".equals(judgementType.getDynamicJudgementHearing().getValue().getLabel())) {
```

Comment:
This is at least the second instance of "No Hearings" in the code now. The code needs refactoring so you make that string a constant. The constant probably belongs in DynamicJudgments

Reply:
Refactored - If this constant becomes used more often (I doubt it), I'll refactor it and add it to ecm-common